content/academy/apify_platform/deploying.md

```diff
@@ -37,9 +52,9 @@ print(add_all_numbers([1, 2, 3, 4])) # -> 10
 </marked-tab>
 ```
 
-> For all sections, we'll have examples for both Node.js and Python.
+> For all lessons in thi sectiom, we'll have examples for both Node.js and Python so that you can follow along in either language.
 
-We've pushed this code to Github and are ready to turn it into an actor, so let's get to it!
+We've pushed this code to Github and are ready to turn it into an actor that takes any number of integers as input, adds them all up, then stores the solution as its output.
```

Comment:
But we haven't really pushed this code to GitHub, no? Where? Do we even need to (for now)?

Reply:
In the context of the lesson we've "pushed it to Github" because we're trying to show them how to take their Github repo and turn it into an actor, right?

Reply:
Ah, I see what you did there. Maybe it's just me but I found it confusing, because I immediately thought I missed or skipped some part. Maybe we could say something like 

No matter if your code lives on your computer or in GitHub, we are ready to turn it into an actor that takes any number of integers as input, adds them all up, then stores the solution as its output.

Am I getting this right?